From 4c3c32dca6c1009c050476fa72fd441058a0bc64 Mon Sep 17 00:00:00 2001
From: "alexey.lysiuk" <alexey.lysiuk@gmail.com>
Date: Sun, 7 Jun 2020 21:52:58 +0300
Subject: [PATCH] - fixed compilation of non-x86 targets
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

src/common/utility/x86.cpp:48:6: error: ambiguating new declaration of ‘void DumpCPUInfo(const CPUInfo*)’
---
 src/common/utility/x86.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/common/utility/x86.cpp b/src/common/utility/x86.cpp
index 983360c09..d674ea24d 100644
--- a/src/common/utility/x86.cpp
+++ b/src/common/utility/x86.cpp
@@ -45,8 +45,9 @@ void CheckCPUID(CPUInfo *cpu)
 	cpu->DataL1LineSize = 32;	// Assume a 32-byte cache line
 }
 
-void DumpCPUInfo(const CPUInfo *cpu)
+FString DumpCPUInfo(const CPUInfo *cpu)
 {
+	return FString();
 }
 #else
 
-- 
2.26.2

